src/main/java/com/stripe/model/Charge.java

```diff
@@ -79,19 +78,53 @@ public void setAmountRefunded(Long amountRefunded) {
 	}
 
 	public String getApplicationFee() {
-		return applicationFee;
+		if (this.applicationFee==null) {
+			return null;
+		}
+		return this.applicationFee.getId();
+	}
+
+	public void setApplicationFee(String applicationFeeID) {
+		if (this.applicationFee==null || (this.applicationFee.isExpanded() && (this.applicationFee.getExpanded().getId() != applicationFeeID))) {
+			this.applicationFee = new ExpandableField<ApplicationFee>(applicationFeeID, null);
+		}
+		this.applicationFee.setId(applicationFeeID);
+	}
+
+	public ApplicationFee getApplicationFeeObject() {
+		if (this.applicationFee==null) {
+			return null;
+		}
+		return this.applicationFee.getExpanded();
 	}
 
-	public void setApplicationFee(String applicationFee) {
-		this.applicationFee = applicationFee;
+	public void setApplicationFeeObject(ApplicationFee c) {
```

Comment:
I just wanted to call out this naming scheme (i.e. suffix `Object`) to anyone else who may be reviewing. I think that this is the right thing to given that we call these "expandable objects" in our API reference, but there may be other alternatives in case there's some disagreement.

Notably, this will be our naming scheme going forward for anything with an expansion. New fields like charge review will have a `getReviewObject` method, but not a `getReview`.